app/com/arpnetworking/metrics/portal/reports/impl/chrome/grafana/BaseGrafanaScreenshotRenderer.java

```diff
@@ -67,8 +69,15 @@ public boolean getIgnoreCertificateErrors(final GrafanaReportPanelReportSource s
     }
 
     @Override
-    public URI getUri(final GrafanaReportPanelReportSource source) {
-        return source.getWebPageReportSource().getUri();
+    public URI getUri(final GrafanaReportPanelReportSource source, final TimeRange timeRange) {
+        try {
+            return new URIBuilder(source.getWebPageReportSource().getUri())
+                    .setParameter("from", Long.toString(timeRange.getStart().getEpochSecond()))
```

Comment:
Unfortunately, I'm not sure that Grafana does the "right thing" with epoch from/to values.

e.g.
`?from=1586787676&to=1586809276`

1586787676 = Monday, April 13, 2020 2:21:16 PM (GMT)
1586809276 = Monday, April 13, 2020 8:21:16 PM  (GMT)

It looks to me that the values are in fact millisecond epochs.

Reply:
![image](https://user-images.githubusercontent.com/1775197/79157955-57c97200-7d8a-11ea-94ad-92cfc2eecbb3.png)

Reply:
Also, this dashboard was configured as "UTC". However, I have seen "bugs" where developers treat epoch as "local". It may be worthwhile testing that if a report were configured as local timezone, that supplying the epoch values still causes it to render the same time interval as if the report were configured UTC (e.g. epoch interpretation is locale agnostic -- if it's not, this quirk will need to be addressed in our plugin ... somehow ... or at least documented)